### PR TITLE
uptycs-185 bug fixes for integration, change playbook inputs and add gatherEvidence Tasks, slight clarification added to parameters section of documentation, uptycs support info added

### PR DESCRIPTION
**Pull request by bkschmoll-uptycs**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR includes:
-bug fixes for integration
-change playbook inputs and add gatherEvidence Tasks
-slight clarification added to parameters section of documentation
-uptycs support info added

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: Playbook input changed from "incident.alertid" to "incident.labels.alertId", addition of the labels path may break backwards compatibility as older Playbooks did not use the labels path in the input.
   - [ ] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Review by guyfreund — CHANGES_REQUESTED** [review]
Overall looking good, but there are two things to do here.
1. Add release notes, see [RN Docs](https://xsoar.pan.dev/docs/integrations/release-notes) for more information.
2. Add fromversion in the integration & playbook (yml files)

**Comment by guyfreund** [comment]
@mayagoldb can you please review the pb?

File: Packs/Uptycs/pack_metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    "author": "Uptycs Inc.",
+    "url": "https://xsoar.pan.dev/docs/reference/integrations/uptycs",
+    "email": "support@uptycs.com",
     "created": "2020-04-14T00:00:00Z",
     "categories": [
         "Analytics & SIEM"
     ],
     "tags": [],
     "useCases": [],
     "keywords": []
```
[the remark this answers is not in the document]
Comment (bkschmoll-uptycs) [reply Packs/Uptycs/pack_metadata.json:15]: I have added the githubUser field to pack_metadata.  We do not currently have a documentation page for integrations on our website.  Can we use the xsoar documentation page for reference until we have added the appropriate documentation to our website and then change the url in a later PR?

File: Packs/Uptycs/pack_metadata.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
     "name": "Uptycs",
     "description": "Fetches data from the Uptycs database.",
-    "support": "xsoar",
+    "support": "partner",
     "currentVersion": "1.0.1",
-    "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
-    "email": "",
+    "author": "Uptycs Inc.",
+    "url": "https://xsoar.pan.dev/docs/reference/integrations/uptycs",
```
[the remark this answers is not in the document]
Comment (bkschmoll-uptycs) [reply Packs/Uptycs/pack_metadata.json:7]: We do not currently have a documentation page for integrations on our website.  Can we use the xsoar documentation page for reference until we have added the appropriate documentation to our website and then change the url in a later PR?

File: Packs/Uptycs/Playbooks/playbook-Uptycs_-_Outbound_Connection_to_Threat_IOC_Incident.yml [hunk 3]
```diff
@@ -3,7 +3,6 @@ version: -1
 name: Uptycs - Outbound Connection to Threat IOC Incident
 description: Get information about connections from IOC incidents.
 starttaskid: "0"
-fromversion: 4.0.0
```
[the remark this answers is not in the document]
Comment (bkschmoll-uptycs) [reply Packs/Uptycs/Playbooks/playbook-Uptycs_-_Outbound_Connection_to_Threat_IOC_Incident.yml:6]: I downloaded the new version of the playbook from my developer instance and I guess 'fromversion' wasn't in it?  I have added it back in.

File: Packs/Uptycs/pack_metadata.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (guyfreund) [reply Packs/Uptycs/pack_metadata.json:7]: The url should be of Uptycs, you can submit a link to the support/contact page of Uptycs.

File: Packs/Uptycs/pack_metadata.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (guyfreund) [reply Packs/Uptycs/pack_metadata.json]: Email should be the support mail, no need for the change.

File: Packs/Uptycs/pack_metadata.json [hunk 4]
```diff
@@ -5,12 +5,13 @@
...
+    "email": "bschmoll@uptycs.com",
     "created": "2020-04-14T00:00:00Z",
     "categories": [
         "Analytics & SIEM"
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "githubUser": "bkschmoll-uptycs"
```
Comment (guyfreund) [inline Packs/Uptycs/pack_metadata.json]:
```suggestion
    "githubUser": ["bkschmoll-uptycs"]
```

File: Packs/Uptycs/pack_metadata.json [hunk 5]
```diff
@@ -5,12 +5,13 @@
     "currentVersion": "1.0.1",
     "author": "Uptycs Inc.",
     "url": "https://xsoar.pan.dev/docs/reference/integrations/uptycs",
-    "email": "support@uptycs.com",
+    "email": "bschmoll@uptycs.com",
```
Comment (guyfreund) [inline Packs/Uptycs/pack_metadata.json:8]:
```suggestion
    "email": "support@uptycs.com",
```

File: Packs/Uptycs/Playbooks/playbook-Uptycs_-_Outbound_Connection_to_Threat_IOC_Incident.yml [hunk 6]
```diff
@@ -3,6 +3,7 @@ version: -1
 name: Uptycs - Outbound Connection to Threat IOC Incident
 description: Get information about connections from IOC incidents.
 starttaskid: "0"
+fromversion: "4.0.0"
```
Comment (guyfreund) [inline Packs/Uptycs/Playbooks/playbook-Uptycs_-_Outbound_Connection_to_Threat_IOC_Incident.yml]: can the fromversion be bumped to 5.0.0?

File: Packs/Uptycs/Integrations/Uptycs/Uptycs.yml [hunk 7]
```diff
@@ -1,3 +1,4 @@
+fromversion: "4.0.0"
```
Comment (guyfreund) [inline Packs/Uptycs/Integrations/Uptycs/Uptycs.yml]: can the fromversion be bumped to 5.0.0?

File: Packs/Uptycs/pack_metadata.json [hunk 8]
```diff
@@ -5,12 +5,13 @@
     "currentVersion": "1.0.1",
```
Comment (guyfreund) [inline Packs/Uptycs/pack_metadata.json:5]: pack version should be the same as the latest rn file.
```suggestion
    "currentVersion": "1.0.2",
```

**Review by guyfreund — CHANGES_REQUESTED** [review]
Looking good, make sure your [build](https://app.circleci.com/jobs/github/demisto/content/97124) is passing.

File: Packs/Uptycs/pack_metadata.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (bkschmoll-uptycs) [reply Packs/Uptycs/pack_metadata.json]: ok, I have changed the url to Uptycs

[the remark this answers is not in the document]
Comment (anara123) [reply Packs/Uptycs/pack_metadata.json]: @bkschmoll-uptycs The URL is not for documentation, but for support. You can provide either email or url or both. These should be contact details, where the users/customers can go in order to submit an issue/ticket that relates to the integration.

**Review by guyfreund — APPROVED** [review]
You should two things to make the CCI pass:
1. update the docker image tag in the integration yml file to `1.0.0.10664` instead of `1.0.0.1126`
2. run demisto-sdk format on the playbook yml file

**Review by guyfreund — CHANGES_REQUESTED** [review]
- Improve the api domain parameter display name:
This needs to be done in the yml file in line 15.
Something like: `display: API Domain (e.g. staging.uptycs.io)`
- Fix the `set_tag` command bug we've seen in the demo.

**Comment by mayagoldb** [comment]
@bkschmoll-uptycs Thanks for the demo!
On the playbook side - 
Add the check for parent process rows after the get-processes command and you're good to go.

**Comment by guyfreund** [comment]
@bkschmoll-uptycs merge content master into your base branch and make sure the working branch is updated too, this will make the build pass.

**Comment by bkschmoll-uptycs** [comment]
> @bkschmoll-uptycs merge content master into your base branch and make sure the working branch is updated too, this will make the build pass.

I have merged content master into my fork, however the build is still not passing.  I believe this may be due to the following error I receive from running the demisto-sdk validate command:

"Value '{'simple': '${inputs.alert_time}'}' is not of type 'text'. Path: '/tasks/1/evidencedata/occurred'.: Path: '/'>"

I'm not sure what should go in this field, the playbook works as intended with the 'occurred' field being filled with the appropriate alert_time from ${inputs.alert_time}.

**Comment by guyfreund** [comment]
@bkschmoll-uptycs we should wait for the demisto-sdk release, it will be done until the end of the week and then the build will pass.

**Comment by guyfreund** [comment]
@bkschmoll-uptycs let's submit the requested changes @mayagoldb and I have asked and then we can merge this PR.

**Review by mayagoldb — APPROVED** [review]

**Comment by bkschmoll-uptycs** [comment]
> @bkschmoll-uptycs let's submit the requested changes @mayagoldb and I have asked and then we can merge this PR.

I believe I have made the requested changes.  Are there any I have missed?

**Comment by guyfreund** [comment]
> > @bkschmoll-uptycs let's submit the requested changes @mayagoldb and I have asked and then we can merge this PR.
> 
> I believe I have made the requested changes. Are there any I have missed?

Yes.
- Improve the API domain parameter display name:
This needs to be done in the yml file in line 15.
Something like: display: API Domain (e.g. staging.uptycs.io)

**Comment by bkschmoll-uptycs** [comment]
> > > @bkschmoll-uptycs let's submit the requested changes @mayagoldb and I have asked and then we can merge this PR.
> > 
> > 
> > I believe I have made the requested changes. Are there any I have missed?
> 
> Yes.
> 
> * Improve the API domain parameter display name:
>   This needs to be done in the yml file in line 15.
>   Something like: display: API Domain (e.g. staging.uptycs.io)

Whoops, forgot about that.  Just pushed.

**Review by guyfreund — APPROVED** [review]

**Pull request merged**